src/utils/history.js

```diff
@@ -59,3 +59,71 @@ export function getStep (item) {
     return SWAP_STATUS_STEP_MAP[item.status]
   }
 }
+
+export const ACTIVITY_FILTER_TYPES = {
+  SWAP: {
+    label: 'Swap',
+    icon: 'swap'
+  },
+  SEND: {
+    label: 'Send',
+    icon: 'send'
+  },
+  RECEIVE: {
+    label: 'Receive',
+    icon: 'receive'
+  }
+}
+
+export const ACTIVITY_FILTER_STATUSES = {
```

Comment:
I think the "FILTER" part in these names is not needed? They are used for multiple purposes and the content of the object is not specific to filtering